app/parsers/bulkrax/csv_parser.rb

```diff
@@ -136,7 +132,12 @@ def create_new_entries
       current_work_ids.each_with_index do |wid, index|
         break if limit_reached?(limit, index)
         new_entry = find_or_create_entry(entry_class, wid, 'Bulkrax::Exporter')
-        Bulkrax::ExportWorkJob.perform_now(new_entry.id, current_run.id)
+        entry = Bulkrax::ExportWorkJob.perform_now(new_entry.id, current_run.id)
+
+        if entry
+          self.headers ||= []
```

Comment:
without setting `headers` as an array first, line 139 would only return `true` instead of the actual headers.

Reply:
that appears to be because you didn't port the initialization line over from the other code =-)

Reply:
just double checked https://github.com/samvera-labs/bulkrax/pull/337/files again. what part did I miss?